bucket/roundedtb.json

```diff
@@ -0,0 +1,34 @@
+{
+    "homepage": "https://github.com/torchgm/RoundedTB/",
+    "version": "R2.2",
+    "license": "GPL-3.0-only",
+    "notes": [
+        "RoundedTB will NOT automatically start when user logs in.",
+        "Open RoundedTB, right-click on the tray icon, and select 'Run on Startup' to add it to startup programs."
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/torchgm/RoundedTB/releases/download/R2.2/RoundedTB.zip",
+            "hash": "6efa70442dc21f281e68595fd94de8652f736a3db612ef1b690eea58e3b6c1a3"
+        }
+    },
+    "uninstaller": {
+        "script": "taskkill /F /IM RoundedTB.exe"
+    },
```

Comment:
Does not make sense. Uninstallation will be stopped if the executable is running

```suggestion
```